### PR TITLE
Resolved responsivness issues

### PR DESCRIPTION
**Pull request by aounhaider1**
Signed-off-by: Aoun Bukhari <bukhari@itemis.de>

### CODE REVIEW

File: extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/dialog.addmodule.html [hunk 1]
```diff
@@ -7,14 +7,14 @@ <h1 ng-show="step==1">Select {{type}} type</h1>
 
 	<form name="myForm">
 
-		<md-input-container class="col-md-12 col-sm-12 configContainer" ng-show="step==2"> <label>Name</label> <input ng-model="name"> </md-input-container>
+		<md-input-container class="col-md-12 col-sm-12 col-xs-12 moduleContainer" ng-show="step==2"> <label>Name</label> <input ng-model="name"> </md-input-container>
```
Comment (watou) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/dialog.addmodule.html]: When using `col-xs-12`, do you need to also specify `col-md-12` and `col-sm-12`?  According to [the docs](https://getbootstrap.com/css/#grid-intro):

> Grid classes apply to devices with screen widths greater than or equal to the breakpoint sizes, and override grid classes targeted at smaller devices. Therefore, e.g. **applying any .col-md-* class to an element will not only affect its styling on medium devices but also on large devices if a .col-lg-* class is not present.**

Comment (aounhaider1) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/dialog.addmodule.html]: Yes, it also seems to work fine without md,sm classes when xs class is present.

Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/dialog.addmodule.html]: @watou Does this answer your question or did you actually want to say that something should be changed here?

Comment (watou) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/dialog.addmodule.html]: The unneeded, larger col-<size>-12 classes could be removed for readability.

**Comment by aounhaider1** [comment]
Pull request updated.

**Comment by kaikreuzer** [comment]
Thanks!

**Pull request merged**